smps.proto

```diff
@@ -15,34 +14,45 @@ message PerformanceTestConfig {
             tcp = 1;
             grpc = 2;
         }
-        
+
```

Comment:
@kanishkarj Don't we need to add `string service_mesh`  ?

Reply:
@leecalcote What is your opinion about it?
The reason I left it out was because service mesh specific details are being encapsulated by `ServiceMeshConfig`, so it felt  redundant. But if it's required I can add it.

Reply:
@kanishkarj I saw the `ServiceMeshConfig` but there is no attribute for identification of which service mesh is defined.

Reply:
Yeah, but the concept was that the service mesh config details would be associated with the result.

Reply:
Unless stringing together(relating) separate yaml files is difficult, I would lean into using foreign keys and avoiding duplicative metadata; making one spec the single source of authority for a given parameter.

Reply:
But we still need a string attribute for service_mesh in the ServiceMeshConfig as of now.

Reply:
Yes totally, I have added it!